frameworks/cassandra/universe/config.json

```diff
@@ -344,6 +344,133 @@
             }
           }
         },
+        "jmx": {
```

Comment:
this code already exists in master on [L220](
https://github.com/mesosphere/dcos-commons/blob/master/frameworks/cassandra/universe/config.json#L220) why are you adding it again?